deployment/Jenkinsfile

```diff
@@ -4,46 +4,34 @@ pipeline {
     stages {
         stage('Checkout') {
             steps {
-                checkout([$class: 'GitSCM', branches: [[name: '*/Dockerimages']], doGenerateSubmoduleConfigurations: false, extensions: [], submoduleCfg: [], userRemoteConfigs: [[url: 'https://github.com/tmarkesic/Auction-App.git']]])
+                checkout([$class: 'GitSCM', branches: [[name: '*/deployment']], doGenerateSubmoduleConfigurations: false, extensions: [], submoduleCfg: [], userRemoteConfigs: [[url: 'https://github.com/tmarkesic/Auction-App.git']]])
                 sh "ls -lart ./*"
             }
         }
-        
-        stage('Build database') {
-	        steps {
-		        dir('./auctionapp/src/main/resource/') {
-			        sh 'docker build -f ../../../../deployment/Dockerfile_DB -t tmarkesic/auction-db .'
-	            }
-	        }	
-	    }
 
-	stage('Build backend') {
-	    steps {
-		    dir('./auctionapp') {
-		         withCredentials([string(credentialsId: 'AWS_ACCESS_ID', variable: 'AWS_ACCESS_KEY'), string(credentialsId: 'AWS_SECRET_ID', variable: 'AWS_SECRET_KEY')]) {			    
+
+	    stage('Build and push backend') {
+	        steps {
+		        dir('./auctionapp') {
+		             withCredentials([string(credentialsId: 'AWS_ACCESS_ID', variable: 'AWS_ACCESS_KEY'), string(credentialsId: 'AWS_SECRET_ID', variable: 'AWS_SECRET_KEY')]) {			    
                     		sh 'chmod +x mvnw'
```

Comment:
Add execute permission to `mvnw` locally and push like that. That should work without needing to change it in Pipeline.